1-exercises/A-accessing-values/exercise3.js

```diff
@@ -21,12 +21,13 @@ let basketballTeam = {
 */
 
 // write code here
-
+const teamPlayers = basketballTeam.topPlayers.sort();
+teamPlayers.forEach((player) => console.log(player));
```

Comment:
can be chained like: `basketballTeam.topPlayers.sort().forEach((player) => console.log(player));`